src/components/utils/http.js

```diff
@@ -0,0 +1,30 @@
+import { events } from 'onfido-sdk-core'
+const SDK_SERVER_URL = 'https://sdk.onfido.com'
+
+const reduceObj = (object, callback, initialValue) =>
+  Object.keys(object).reduce(
+    (accumulator, key) => callback(accumulator, object[key], key, object),
+    initialValue)
+
+const objectToFormData = (object) =>
+  reduceObj(object, (formData, value, key) => {
+    formData.append(key, value)
+    return formData;
+  }, new FormData())
+
+export const postToServer = (payload, serverUrl, token, onComplete) => {
+  const request = new XMLHttpRequest()
+  const url = serverUrl ? serverUrl : SDK_SERVER_URL
+  request.open('POST', `${url}/validate_document`)
+  request.responseType = 'json'
+  request.setRequestHeader('Content-Type', 'application/json')
+  request.setRequestHeader('Authorization', token)
+
+  request.onload = () => {
```

Comment:
It might be worth adding and on error callback in case CORS or the server are down: http://stackoverflow.com/questions/10584318/when-should-xmlhttprequests-onerror-handler-fire